types/rollup/index.d.ts

```diff
@@ -24,12 +24,12 @@ export interface Warning {
 export interface BundleOptions {
 	/** The format of the generated bundle. */
 	format?: Format
```

Comment:
you missed this deprecation:

https://github.com/rollup/rollup/blob/b949eb08169115ff66648838cbc4833379bf9440/src/utils/deprecateOptions.js#L60